snyk/snyk
# fix: exclude application vulns from filtering

#### What does this PR do?

Currently if using `--file` and `--exclude-base-image-vulns` flag would filter
out all the vulnerabilities including application vulnerabilites which is not
related to the base image. We want to only filter out the base image vulnerabilites.

#### Where should the reviewer start?

src/lib/snyk-test/legacy.ts

#### How should this be manually tested?

* npm run build
* build an image with this project
* `snyk container test <image-has-been-built> --app-vulns --file=Dockerfile --exclude-base-image-vulns; echo $?`
* exit code should be 1 instead of 0

#### Any background context you want to provide?

[zendesk ticket](https://snyk.zendesk.com/agent/tickets/16119)
